modules/Microsoft.ServiceFabric/clusters/.test/min/deploy.test.bicep

```diff
@@ -0,0 +1,57 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'ms.servicefabric.clusters-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'sfcmin'
+
+// =========== //
+// Deployments //
+// =========== //
+
+// General resources
+// =================
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+  name: resourceGroupName
+  location: location
+}
+
+// ============== //
+// Test Execution //
+// ============== //
+
+module testDeployment '../../deploy.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name)}-test-${serviceShort}'
+  params: {
+    name: '<<namePrefix>>${serviceShort}001'
+    managementEndpoint: 'https://<<namePrefix>>${serviceShort}001.westeurope.cloudapp.azure.com:19080'
+    reliabilityLevel: 'None'
+    nodeTypes: [
```

Comment:
It looks from the readme the only 2 required parameters are name and managementEndpoint, so I'd suggest to remove the rest and rerun.

Reply:
The parameter reliabilityLevel is described as Option but it does not have fefault vale:
`@description('Optional. The reliability level sets the replica set size of system services. Learn about ReliabilityLevel (https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-capacity). - None - Run the System services with a target replica set count of 1. This should only be used for test clusters. - Bronze - Run the System services with a target replica set count of 3. This should only be used for test clusters. - Silver - Run the System services with a target replica set count of 5. - Gold - Run the System services with a target replica set count of 7. - Platinum - Run the System services with a target replica set count of 9.')
param reliabilityLevel string`

Should I put it as Required or set default no None?

Reply:
Good question 😀 I guess it would be one of the settings part of the secure by default topic. Maybe PSRule could help us with that. For the time being, since I'm not experienced with the service, I'd tend to put the value as Required in the module.

Reply:
Set to REquired and removed non required parameters from this test

Reply:
nodeTypes is mandatory not optional as pointed in the deploy.bicep